app/lib/theme/widgets/ReadOnlyAdjustableCalculatedValueWidget.tsx

```diff
@@ -57,21 +38,19 @@ const ReadOnlyAdjustableCalculatedValueWidget: React.FC<WidgetProps> = (
           ) : (
             <em>Not added</em>
           )}
+          <style jsx>{`
+            .adjustedValue {
+              display: flex;
+              flex-basis: 100%;
+              padding-top: 0.5em;
```

Comment:
Overall the new styling is a huge improvement so I'd be comfortable approving this as-is, but I did come across some extra spacing when the adjusted value is filled out if you want to keep tweaking:
![Screenshot from 2023-07-21 15-43-34](https://github.com/bcgov/cas-cif/assets/77306764/b32b7c74-0243-4cba-bf59-73bb3535f4ad)
![Screenshot from 2023-07-21 15-47-21](https://github.com/bcgov/cas-cif/assets/77306764/2312c399-7c3a-407c-a5f9-ee9bf7b46559)

Reply:
Done